File: 11_November_Batch/C/fibonacci.c
#include <stdio.h>
void fibonacci(int n) {
    if (n == 1) {
        printf("0\n");
    } else if (n == 2) {
        printf("0\n");
        printf("1\n");
    } else {
        printf("0\n");
        printf("1\n");
        n = n - 2; 
        int a = 0, b = 1, c;
        while (n!=0) {
            c = a + b;
            printf("%d\n", c);
            a = b;
            b = c;
            n--;
        }
    }
}

int main() {
    int n;
    printf("Enter the number of terms: ");
    scanf("%d", &n);
    fibonacci(n);
    return 0;
}

File: 4_August_Batch/Java/Fibonacci.java
import java.util.*;
public class Fibonacci{
    public static void printFibonacci(int a){
        System.out.println("Fibonacci Series: ");
        if (a==1){
            System.out.println(0);
        }
        else if(a==2){
            System.out.println(0);
            System.out.println(1);
        }
        else{
            a=a-2;
            System.out.println(0);
            System.out.println(1);
            int n=0,m=1,o;
            while(a!=0){
            o=n+m;
            System.out.println(o);
            n=m;
            m=o;
            a--;
            }

        }
    }
    public static void main(String args[]){
        Scanner sc= new Scanner(System.in);
            System.out.print("Enter number of terms: ");
            int a = sc.nextInt();
            printFibonacci(a);

    }
}